tests/kernel/mem_protect/futex/src/main.c

```diff
@@ -447,14 +447,14 @@ void test_main(void)
 {
 	ztest_test_suite(test_futex,
 			 ztest_user_unit_test(test_user_futex_bad),
-			 ztest_unit_test(test_futex_wait_forever_wake),
-			 ztest_unit_test(test_futex_wait_timeout_wake),
-			 ztest_unit_test(test_futex_wait_nowait_wake),
-			 ztest_unit_test(test_futex_wait_forever_wake_from_isr),
-			 ztest_unit_test(test_futex_multiple_threads_wait_wake),
-			 ztest_unit_test(test_multiple_futex_wait_wake),
-			 ztest_unit_test(test_futex_wait_forever),
-			 ztest_unit_test(test_futex_wait_timeout),
-			 ztest_unit_test(test_futex_wait_nowait));
+			 ztest_1cpu_unit_test(test_futex_wait_forever_wake),
+			 ztest_1cpu_unit_test(test_futex_wait_timeout_wake),
+			 ztest_1cpu_unit_test(test_futex_wait_nowait_wake),
+			 ztest_1cpu_unit_test(test_futex_wait_forever_wake_from_isr),
+			 ztest_1cpu_unit_test(test_futex_multiple_threads_wait_wake),
+			 ztest_1cpu_unit_test(test_multiple_futex_wait_wake),
+			 ztest_1cpu_unit_test(test_futex_wait_forever),
+			 ztest_1cpu_unit_test(test_futex_wait_timeout),
+			 ztest_1cpu_unit_test(test_futex_wait_nowait));
```

Comment:
Note that the 1cpu feature halts one CPU by grabbing a lock on it and spinning, which is fine from a correctness perspective but a performance hassle when run under sanitycheck (it doubles the load, and makes timer-related spurious failures more likely on other tests running on the host).  In situations like this where basically everything is MP-unsafe it's probably better to use CONFIG_MP_NUM_CPUS=1 instead (or in addition to -- 1cpu obviously is a noop if there's only one cpu)